File: list/ans40.cpp
#include <iostream> 
#include <vector>
#include <string>


int main(){
    std::vector <std::string> data ={
        "be","have","do","say","get","make","go","know","take","see","come",
        "think","look","want","give","use","find","tell","ask","work","seem",
        "feel","leave","call"};

    
    for(int i = 0; i<data.size(); i++){
        for(int j = 0; j<data.size()-i-1; j++){
            char begin = data[j][0]; 
            char end   = data[j+1][0];
            if(begin > end){
                // std::string first = data[j]; 
                // data[j] = data[j+1]; 
                // data[j+1] = first;
                std::swap(data[j], data[j+1]);
            }
        }
    }

    std::cout << "{"; 
    for(int i = 0; i<data.size(); i++){
        std::cout << data[i];
        if(i != data.size()-1){
            std::cout << ", "; 
        }
    }
    std::cout << "}"<<std::endl;


}

File: list/ans40.py
strs = ['be','have','do','say','get','make','go','know','take','see','come','think','look','want','give','use','find','tell','ask','work','seem','feel','leave','call']



for i in range(0, len(strs)):
    for j in range(0, len(strs)-i-1):
        if strs[j][0] > strs[j+1][0]:
            strs[j], strs[j+1] = strs[j+1], strs[j]

print(strs)

data = [1,'a']
print(data)